stanford_person.install

```diff
@@ -25,4 +25,9 @@ function stanford_person_install() {
   if (!\Drupal::service('module_handler')->moduleExists('default_content')) {
     _stanford_person_install_default_photo();
   }
+  else {
+    // Just put the file in place.
+    _stanford_person_install_default_photo_file();
```

Comment:
oh so this is for when the default content json is consumed, the image isn't created so we have to put it there ourself.

Reply:
Yeah, that was my learning moment as well. Got tricked by the fact my development environment had the image already there.